website_sale_product_assortment/controllers/website_sale.py

```diff
@@ -17,15 +17,22 @@ def _get_products_allowed(self):
             .search(
                 [
                     ("is_assortment", "=", True),
-                    ("website_availability", "=", "no_show"),
                     "|",
                     ("website_ids", "=", False),
                     ("website_ids", "=", website_id),
                 ]
             )
         )
+        no_restriction_assortments = any(
+            assortment.website_availability == "no_show" for assortment in assortments
+        )
+
```

Comment:
Please remove extra empty lines inside a method.

Reply:
Done! Thanks for the correction